src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIZE = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
+    ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
please do not forget about access modifiers
```suggestion
    private final ColorSupplier colorSupplier = new ColorSupplier();
```